test/mesh.jl

```diff
@@ -172,7 +172,7 @@
     @test measure(grid) ≈ T(1)
     @test centroid(grid, 1) == P2(0.1, 0.05)
     @test centroid(grid[1]) == P2(0.1, 0.05)
-    @test centroid(grid, 2) == P2(0.3, 0.05)
+    @test centroid(grid, 2) ≈  P2(0.3, 0.05)
     @test centroid(grid[2]) == P2(0.3, 0.05)
```

Comment:
Given that we are changing one line, maybe we should change the entire block for consistency.